cloud/google/configs/ct-mirror-certly1.sh

```diff
@@ -0,0 +1,9 @@
+export INSTANCE_TYPE="mirror"
+export PROJECT="ct-mirror-certly1"
```

Comment:
oh looks like you're doing it on the basis of the _1.pem file, and you do the same below.
What's the story there?